doc/mgr/dashboard.rst

```diff
@@ -378,6 +377,31 @@ The available iSCSI gateways must be defined using the following commands::
 Enabling the Embedding of Grafana Dashboards
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+`Grafana`_ requires data from Prometheus. Although Grafana can use other data
+sources, the Grafana dashboards we provide contain queries that are specific to
+`Prometheus <https://prometheus.io/>`_. Our Grafana dashboards therefore
```

Comment:
Minor nit: move the URL to the first mentioning of Prometheus in Line 380 instead.